notebook/static/notebook/js/quickhelp.js

```diff
@@ -30,7 +30,7 @@ define([
     var platform_specific;
 
     if (platform === 'MacOS') {
-        // Mac OS X specific
+        // Mac OS X (MacOS) specific
```

Comment:
Let's avoid MacOS. Spaces should always be between Mac, OS, X, and if we want a short name, OS X or Mac would be the right choice. It hasn't officially been "Mac OS X" since 2012.